instagram-app/src/components/CommentSection.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { Input } from 'reactstrap'
+
+const CommentSection = props => {
+    console.log(props.comments)
+    return (
+        <div>
+            {props.comments.map(comments => (
```

Comment:
Good work accessing the comments from the data. This is a good introduction into accessing nested data.